index.html

```diff
@@ -249,7 +249,7 @@ <h2>Facebook 360</h2>
         </div>
         <div class="tonic-desktop1">
           <div class="cont-right1">
-            <div class="imagez1"><img src="images/story.png" alt="snap" /></div>
+            <div class="imagez1"><img src="images/desk2.svg" alt="snap" /></div>
```

Comment:
Kindly update the alt text so that it conveys the purpose of the respective image as described above 😃